core/bindings.rs

```diff
@@ -1471,7 +1389,7 @@ fn is_proxy(
   rv.set(v8::Boolean::new(scope, args.get(0).is_proxy()).into())
 }
 
-fn throw_type_error(scope: &mut v8::HandleScope, message: impl AsRef<str>) {
+pub fn throw_type_error(scope: &mut v8::HandleScope, message: impl AsRef<str>) {
```

Comment:
This is code smell - it introduces circular dependency between `deno_core/` and `ops/`

Reply:
No it's fine,`deno_ops` is completely compile-time but the code it generates may need to call these helpers and all op code inherently depends on deno_core so nothing abnormal there